source/configure/configuation-in-a-database.rst

```diff
@@ -1,215 +0,0 @@
-Configuration in a database
```

Comment:
Please revert this file deletion, then replace the introductory sentence to: "You can use the database as the single source of truth..."